Initial design doc (#1)
danzziy/Tic-Tac-GO: README.md

# Tic-Tac-GO

Tic-Tac-GO is a website with a GO backend and a Javascript frontend where people can play 

> tic-tac-toe against there friends, family, and random people online. The vision for this site is to

Author: ```suggestion
tic-tac-toe against their friends, family, and random people online. The vision for this site is to
```

> tic-tac-toe against their friends, family, and random people online. The vision for this site is to
> have users who enter the website be presented with two choices; to play with a friend or play with

Author: ```suggestion
tic-tac-toe against their friends, family, and random people online. When users enter the site, they will be presented with two choices; to play with a friend or play with
```
